src/routes/upload/index.ts

```diff
@@ -81,6 +80,68 @@ router.post('/form/records/:id', async (req: any, res) => {
   }
 });
 
+/**
```

Comment:
@Hugo-Plissonneau something we should start to do, is to put description of all the new methods.
You can do it using typing /** on top of the method, it should automate something in VScode to show placeholder for more comments ( including all the arguments of the method )